app/models/solidus_avatax_certified/line.rb

```diff
@@ -5,11 +5,13 @@ class Line
     attr_reader :order, :lines
     include Spree::Tax::TaxHelpers
 
-    def initialize(order, invoice_type, refund = nil)
+    def initialize(order, invoice_type, refund = nil, override_tax = nil)
       @order = order
       @invoice_type = invoice_type
       @lines = []
       @refund = refund
+      @refund_reason = refund&.reason
```

Comment:
Does this mean `@refund_reason` could be `nil`?
Will `@refund_reason.name` below throw error then?

Reply:
same model is used while order submission too and not just for refunds. `@refund` and `@override_tax` will be nil then and so will be `refund_reason`. But `@refund_reason.name` will not be called in that case as we do 
`return {} unless @override_tax` in `tax_override_hash` method. So `tax_override_hash` will return empty hash for order tax request and for refunds other than our 4 refund cases of interest

Reply:
👍